verifier/trustpolicy/trustpolicy.go

```diff
@@ -263,22 +267,29 @@ func LoadDocument() (*Document, error) {
 	policyDocument := &Document{}
 	err = json.NewDecoder(jsonFile).Decode(policyDocument)
 	if err != nil {
-		return nil, err
+		if errors.Is(err, io.EOF) {
+			return nil, errors.New("empty trustpolicy.json file. See a valid trustpolicy.json example at https://github.com/notaryproject/notaryproject/blob/main/specs/trust-store-trust-policy.md#trust-policy")
+		}
+		return nil, errors.New("invalid trustpolicy.json file. See a valid trustpolicy.json example at https://github.com/notaryproject/notaryproject/blob/main/specs/trust-store-trust-policy.md#trust-policy")
```

Comment:
Instead of `main`, we should use a fixed version tag.

Reply:
I replaced it with `v1.0.0-rc.1`.